src/utils/status.ts

```diff
@@ -22,6 +22,10 @@ export class Status implements StatusBlueprint {
     }
 
     buildMethod(type: keyof StatusBlueprint): StatusMethod {
+        if (process.env.NODE_ENV?.toLowerCase() === 'test') {
+            // TODO: use determineNodeEnv() here
+            return () => {} // eslint-disable-line @typescript-eslint/no-empty-function
+        }
```

Comment:
I get the frustration with this, though sometimes when coding locally logging is good to have. It's somewhat unintuitive to find this piece of code to re-enable it. 

I think there's on alternative to this, adding [`LOG_LEVEL=none`](https://github.com/PostHog/plugin-server/blob/b3dbc21dc6574fc129d0d492c750c56ad03554aa/src/config/config.ts#L50) as a default for test mode.

It might have a bug, in that the existing console patching mechanism will also swallow messages sent by jest that would be good to still see. If so, the solution would be to just add this log level filtering inside `status`.

Reply:
Eh, removed this path altogether as it had circular import problems with `config`, maybe another PR